test/lib/ufe/CMakeLists.txt

```diff
@@ -113,5 +104,9 @@ foreach(script ${test_script_files})
         "MAYA_PLUG_IN_PATH=${mayaPluginPath}"
         "PXR_PLUGINPATH_NAME=${CMAKE_INSTALL_PREFIX}/lib/usd"
         "MAYA_NO_STANDALONE_ATEXIT=1"
+        "UFE_PREVIEW_VERSION_NUM=${UFE_PREVIEW_VERSION_NUM}"
```

Comment:
Adding this env var allows the test to have conditional code based on the UFE version.